src/app/pages/poll/poll.component.ts

```diff
@@ -39,6 +42,7 @@ export class PollComponent {
   pollStatus = 1; // 1 = Planned; 2 = Running; 3 = Finished
   showEditPollDialog = false;
   showEditPollItemDialog = false;
+  selectedPollItem: MultipleChoiceItemParticipant|QuizItemParticipant|OpenTextItemParticipant;
```

Comment:
Is automatic formatting not adding a space between `|`?

Reply:
Nope